README.md

```diff
@@ -10,4 +10,30 @@ What if...
 
 ...it had better transfer and caching properties than actual websites?
 
-...you could host the whole database in a plain-old browser tab?
\ No newline at end of file
+...you could host the whole database in a plain-old browser tab?
+
+## Contributing
+
+Prerequisites:
+
+- You must have the [Radicle CLI](https://radicle.xyz/download) installed (`rad` command)
```

Comment:
More or less the same comment as https://github.com/dialog-db/dialog-db/pull/64/files#r2096558870